packages/main/bundle.common.js

```diff
@@ -87,6 +87,8 @@ import StandardListItem from "./dist/StandardListItem.js";
 import CustomListItem from "./dist/CustomListItem.js";
 import GroupHeaderListItem from "./dist/GroupHeaderListItem.js";
 
+Icon.suppressNotImportedIconsWarning();
```

Comment:
instead of suppressing the default behaviour (with additional code), let's change the default so that the icon bundle is never imported transitively (this will be breaking).

Then whoever uses an icon without an import will see the error and can decide whether to import individual icons or add a global import on this line of the icon bundle.